src/packages/client/src/__tests__/integration/errors/multiple-errors-one-client/test.ts

```diff
@@ -0,0 +1,70 @@
+import { getTestClient } from '../../../../utils/getTestClient'
+
+test('multiple-errors-one-client', async () => {
+  const PrismaClient = await getTestClient()
+  const prisma = new PrismaClient()
+
+  try {
+    await prisma.user.findMany({ invalidArg: true })
+  } catch (e) {
+    console.log(e)
+    expect(e).toMatchInlineSnapshot(`
+
+      Invalid \`prisma.user.findMany()\` invocation in
+      /client/src/__tests__/integration/errors/multiple-errors-one-client/test.ts:8:23
+
+         5 const prisma = new PrismaClient()
+         6 
+         7 try {
+      →  8   await prisma.user.findMany({
```

Comment:
This should be different per https://github.com/prisma/prisma/issues/7874, but somehow the error does not happen here via Jest. Weird.